PhysicsTools/Utilities/interface/RootMinuitCommands.h

```diff
@@ -126,27 +126,34 @@ namespace fit {
     using namespace std;

```

Comment:
I can, however that using is within the scope of a function which is allowed by our coding conventions.